package.json

```diff
@@ -11,15 +11,20 @@
   "scripts": {
     "start": "node app.js",
     "pretest": "npm run lint",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:e2e": "echo 'no e2e tests available.'",
+    "test": "npm run test:unit",
+    "test:e2e": "npm run selenium:install && npm run selenium:start && npm run nightwatch",
```

Comment:
Okay, I've decided that the most straightforward way to handle this with the least pain is to, as you suggested above, add this to the contrib.  I don't want to add a bunch of instrumentation to the node run test:e2e job for something that really isn't going to happen all that often